### DISCUSSION TITLE
"Give guests items" to be added to plugin API

### DISCUSSION THREAD

**Discussion by Manticore-007** [OpenRCT2/OpenRCT2#16541] [Feature requests] [answered]
Now there is the possibility to give all guests balloons and park maps in the cheats menu. I would love to have the ability to give guests items; like hats, sunglasses, food/drinks etc. I want to add this feature to my PeepEditor plugin.

**Comment by KatieZeldaKat** [comment]
Created a mock-up of what the API would look like in `openrct2.d.ts` in #21062. Doesn't have any functionality yet, as I was just trying to see how straightforward I could make the methods and variables for plugin devs.

**Reply by sjoerddebruin** [reply, marked as answer]
The above has been merged in January 2024.
